packages/client/src/pages/IntroPage.tsx

```diff
@@ -2,28 +2,25 @@ import React, { Suspense } from "react";
 import { Helmet } from "react-helmet";
 import { useHistory } from "react-router-dom";
 
-import Layout from "../components/Layouts/DefaultLayout";
+import Layout from "../components/Layouts/TopicLayout";
 import Loading from "../components/Loading";
 import Nav from "../components/Nav";
-import { Topic } from "../config";
 import { actions, sections } from "../config/matomo";
-import withChecker from "../hoc/withChecker";
-import withTracking, { MatomoTrackEventProps } from "../hoc/withTracking";
+import { useChecker, useTopic, useTracking } from "../hooks";
 import { geturl, routes } from "../routes";
+import RedirectPage from "./RedirectPage";
 
-export type IntroPageProps = {
-  checker?: any;
-  topic: Topic;
-};
-
-const IntroPage: React.FC<IntroPageProps & MatomoTrackEventProps> = ({
-  checker,
-  matomoTrackEvent,
-  topic,
-}) => {
+const IntroPage: React.FC = () => {
   const history = useHistory();
+  const { matomoTrackEvent } = useTracking();
+  const checker = useChecker();
```

Comment:
It seems this file doesn't need `checker`. checker is passed as prop to Intro:
`<Intro checker={checker} />`
So the Intro file should need the hook instead of this one

Reply:
Ah good one. I'll check it out.

Reply:
Intro's now all should implement IntroProps 👍